tests/Unit/Libraries/Cms/Image/ImageTest.php

```diff
@@ -1431,6 +1431,154 @@ public function testSanitizeOffset($input, $expected)
 		);
 	}
 
+	/**
```

Comment:
If you have refactored/reworked your Images file you can remove the duplicate code from this file and see which tests you really need.